Sourcery/Parsing/Composer.swift

```diff
@@ -25,6 +25,17 @@ struct Composer {
         let types = parserResult.types
         let typealiases = self.typealiasesByNames(parserResult)
 
+        //set definedInType for all methods and variables
+        types
+            .forEach { type in
+            type.variables.forEach { variable in
+                variable.definedInType = type
```

Comment:
Should we do it for types too? How does it relate to containedTypes/parent property? Do we resolve parent type properly when typealias is used?